docs/source/how_to_guides/configure_cloud_storage_cred.md

```diff
@@ -0,0 +1,101 @@
+# Configure Cloud Storage Credentials
+
+Streaming dataset supports the following cloud storage providers to stream your data directly to your instance.
+- [Amazon S3](https://aws.amazon.com/s3/)
+- [Google Cloud Storage](https://cloud.google.com/storage)
+- [Oracle Cloud Storage](https://www.oracle.com/cloud/storage/)
+
+## Amazon S3
+
+For S3 bucket with a public access, no additional setup is required, simply specify the S3 URI of the resource.
+
+### MosaicML platform
+
+For [MosaicML platform](https://www.mosaicml.com/cloud) users, follow the steps mentioned in the [Amazon S3](https://mcli.docs.mosaicml.com/en/latest/secrets/s3.html) doc on how to configure the cloud provider credentials.
+
+### Others
+
+First, make sure the `awscli` is installed, and then run `aws configure` to create the config and credential files:
+
+```
+python -m pip install awscli
+aws configure
+```
+
+Note: the requested credentials can be retrieved through your [AWS console](https://aws.amazon.com/console/), typically under “Command line or programmatic access”.
+
+Your config and credentials files should follow the standard structure output by `aws configure`:
+
+`~/.aws/config`
+
+```
+[default]
+region=us-west-2
+output=json
+
+```
+
+`~/.aws/credentials`
+
+```
+[default]
+aws_access_key_id=AKIAIOSFODNN7EXAMPLE
+aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+
+```
+
+More details on these files can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html). The streaming dataset reads the `~/.aws/config` and `~/.aws/credentials` to authenticate your credentials and stream data into your instance.
+
+## Google Cloud Storage
+
+### MosaicML platform
+
+For [MosaicML platform](https://www.mosaicml.com/cloud) users, follow the steps mentioned in the [Google Cloud Storage](https://mcli.docs.mosaicml.com/en/latest/secrets/gcp.html) doc on how to configure the cloud provider credentials.
```

Comment:
After the link put MCLI so that it reads MCLI doc on how to configure...

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/169/commits/c164c1329e2a1cdfc5d49ece7647e05bca8ce411.